src/components/Footer.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import logo from '../assets/1logo.png'
+import { FaLinkedin } from "react-icons/fa6";
+import { RiInstagramFill } from "react-icons/ri";
+import { BsDiscord } from "react-icons/bs";
+
+export default function Footer() {
+  return (
+    <>
+    <div className='mt-24  border-t-2 border-gray-400 font'>
+      <div className='flex mt-3 items-center justify-between mx-3'>
+         <img src={logo} alt="footer" className='lg:w-[120px] w-[50px] md:w-[80px]' />
```

Comment:
Footer image is too big 
![CleanShot 2024-02-18 at 21 38 29@2x](https://github.com/khattakdev/dope-devs/assets/37709578/251f3058-cc15-4cb1-a82b-75a662888a10)